### ISSUE TITLE
RBS constant in version.rb file gives untyped

### ISSUE THREAD

**Issue by sandelius** [AaronC81/sord#145]
**Describe the bug**
When running sord with: `bundle exec sord --rbs types.rbs` the `VERSION` constant gets untyped. 

**To Reproduce**
``` 
$ bundle gem my_gem && cd my_gem
$ bundle add sord
$ bundle install
# add # @return [String] to the constant in `lib/my_gem/version.rb`
$ bundle exec sord --rbs types.rbs
```
**Expected behavior**
``` 
module MyGem
  VERSION: String
end
```

**Actual behavior**
``` 
module MyGem
  VERSION: untyped
end
```

**Additional information**
Made the steps above for you https://github.com/iamsandelius/sord-version-constant

Generated .rbs is in `types.rbs`

I suspect it has to do with the module being defined twice but that's just swinging blind without diving deeper into it.

**Comment by matmorel** [comment]
It has been fixed by #130, you can use master as a workaround:

```ruby
# Gemfile
source "https://rubygems.org"
gem "sord", git: "https://github.com/AaronC81/sord", branch: "master"
gemspec
```

```sh
bundle
bundle exec sord --rbs types.rbs
```

```rbs
# types.rbs
module MyGem
  VERSION: String
end
```

Unfortunately this can't be placed in the `my_gem.gemspec` file until the next release.

**Comment by iamsandelius** [comment]
@matmorel ohh I saw that PR and also that it was merge so long ago that I just assumed it was released after that. Thx.

**Comment by AaronC81** [comment]
Sorry for the delay, everyone - just released Sord 4.0.0 to RubyGems, which includes this fix.

I'll close this issue since that should fix it without needing to use `master`, but please ping me if you're still having problems, and I'll get it sorted!
